geps/gep-1619.md

```diff
@@ -210,18 +666,157 @@ TBD - See [open question](#open-questions) about session affinity API
 - Should we include both session persistence and session affinity API configurations in this GEP or just focus on session persistence?
 - Should both session persistence and session affinity be designed as common load balancing policy API? Should they be configured as separately in the API?
 - What happens when session persistence is broken because the backend is not up or healthy? If that's an error case, how should that be handled? Should the API dictate the http error code? Or should the API dictate fall back behavior?
+- What happens when session persistence causes traffic splitting scenarios to overload a backend?
```

Comment:
This is really the application developer+cluster admin's problem. Note that session persistence is needed for correct functionality so the traffic splitting (or otherwise session load balancing) should be properly implemented to prevent overloading a backend and I think the Gateway API collectively should have the functionality to achieve that

Reply:
That's a good point. Still probably worth a call-out in this GEP. Like the last comment, I've taken a note, but address this in another PR.